app/models/player.rb

```diff
@@ -1,3 +1,7 @@
 class Player < ActiveRecord::Base
   belongs_to :team
-end
+
+  def age
+    (Date.today.strftime('%Y%m%d').to_i - date_of_birth.strftime('%Y%m%d').to_i) / 10000
```

Comment:
`10000` is a bit of a mystery. Consider making it a constant with a name that explains its purpose. See article on [magic numbers](https://eliotsykes.com/magic-numbers) if you get a chance.